src/Context/CommonContext.php

```diff
@@ -384,41 +429,29 @@ public function iShouldNotSeeElementByCss($selector): void
         $this->assertSession()->elementNotExists('css', $selector);
     }
 
-    /**
-     * @Then I should see :url page url
-     * @param string $url
-     * @throws \Exception
-     */
-    public function thenIShouldSeePage(string $url): void
```

Comment:
Are we sure that it is safe to remove this context method? This is a library and there might be applications testing functionality depending on this code.

Standard practice to remove code from libraries is: marking the methods as `@deprecated` and release a new major version with all deprecated methods removed and BC upgrade path and changes documented.

Reply:
@ayanozturk I have added context method, Venu said its duplicate and few tests are using those method in SR and CR projects. So I have to change those once it merged.

Reply:
Agree with @ayanozturk here, we should mark it as deprecated and remove in the next major version.